Networking/Extensions/DecodedExtension.swift

```diff
@@ -29,6 +29,17 @@ public extension Decoded {
     
 }
 
+public extension Decodable where Self: RawRepresentable, Self.RawValue == String {
```

Comment:
I don't get this. Those type already know how to decode. What else do you think we should add here?